File: 백준/Bronze/16017. Telemarketer or not？/Telemarketer or not？.py
import sys
input=lambda:sys.stdin.readline().rstrip()
arr=[int(input()) for _ in range(4)]
if (arr[0]==8 or arr[0]==9) and (arr[3]==8 or arr[3]==9):
    if arr[1]==arr[2]:
        print("ignore")
    else:
        print("answer")
else:
    print("answer")

File: 백준/Bronze/16017. Telemarketer or not？/Telemarketer or not？.cc
#include <iostream>
#include <vector>
using namespace std;
int main(void) {
    vector<int> arr(4);
    for (int i = 0; i < 4; i++)
        cin >> arr[i];
    if ((arr[0] == 8 || arr[0] == 9) && (arr[3] == 8 || arr[3] == 9)) {
        cout << (arr[1] == arr[2] ? "ignore" : "answer");
    } else {
        cout << "answer";
    }
    return 0;
}